flow/src/benchmarking.rs

```diff
@@ -1,16 +1,148 @@
 //! Benchmarking setup for gamedao-flow
 use super::*;
 
-#[allow(unused)]
 use crate::Pallet as Flow;
-use frame_benchmarking::{account, benchmarks, impl_benchmark_test_suite};
+use frame_benchmarking::{account, benchmarks, impl_benchmark_test_suite, whitelisted_caller};
 use frame_system::RawOrigin;
-use sp_std::vec;
+use frame_support::traits::Hooks;
+use sp_runtime::{DispatchError, traits::{Saturating}};
+use sp_std::{vec};
+
+
+fn fund_account<T: Config>(account_id: &T::AccountId, multiplier: Option<u32>) -> Result<(), DispatchError> {
```

Comment:
```suggestion
fn fund_account<T: Config>(account_id: &T::AccountId, amount: T::Balance) -> Result<(), DispatchError> {
```
Or hardcoding the value also could work. Having a multiplier as an argument makes logic not so straightforward.